src/eva/vision/data/datasets/segmentation/total_segmentator.py

```diff
@@ -134,19 +123,21 @@ def __len__(self) -> int:
         return len(self._indices) * self._n_slices_per_image
 
     @override
-    def load_image(self, index: int) -> np.ndarray:
+    def load_image(self, index: int) -> tv_tensors.Image:
         image_path = self._get_image_path(index)
         slice_index = self._get_sample_slice_index(index)
         image_array = io.read_nifti_slice(image_path, slice_index)
-        return image_array.repeat(3, axis=2)
+        image_rgb_array = image_array.repeat(3, axis=2)
```

Comment:
what is the `.repeat(3` here for?

Reply:
its because is a grey image and basically make it 3-channeled; maybe its better to use cv2 function though

Reply:
ah yeah just remembered why we are not using cv2; its because the image is in uint16 and the cv2 function does not work with these formats

Reply:
I see, and why do we need 3 channels?

Nit: Here we use `axis=2` to access the channel dimension, and below in the code we use `axis=-1`